packages/gatsby/src/utils/worker/child/queries.ts

```diff
@@ -11,16 +11,30 @@ import { buildSchema } from "./schema"
 import {
   IAddPendingPageDataWriteAction,
   ICreatePageDependencyAction,
+  IGatsbyState,
   IPageQueryRunAction,
   IQueryStartAction,
 } from "../../../redux/types"
+import { DeepPartial } from "redux"
 
 export function setComponents(): void {
   setState([`components`, `staticQueryComponents`])
 }
 
 export function saveQueries(): void {
```

Comment:
I know it's not new in this PR, but maybe we could rename this function to be more explicit (`saveQueriesDependencies` or something to indicate that we are storing query deps)? For a second I was a bit confused reading `runQueriesInWorkersQueue` (even tho I knew what is supposed to happen :) )